astropy/nddata/utils.py

```diff
@@ -494,7 +494,7 @@ class Cutout2D:
         `~astropy.coordinates.SkyCoord`, in which case ``wcs`` is a
         required input.
 
-    size : int, array-like, `~astropy.units.Quantity`
+    size : int, array_like, `~astropy.units.Quantity`
```

Comment:
To be consistent with the other ones I see in diff.
```suggestion
    size : int, array_like, or `~astropy.units.Quantity`
```